src/components/application_manager/test/application_manager_impl_test.cc

```diff
@@ -836,8 +857,8 @@ TEST_F(ApplicationManagerImplTest,
 
 TEST_F(ApplicationManagerImplTest,
        OnDeviceSwitchingFinish_ExpectUnregisterAppsInWaitList) {
-  std::shared_ptr<MockApplication> switching_app_ptr =
-      std::make_shared<MockApplication>();
+  std::shared_ptr<NiceMock<MockApplication> > switching_app_ptr =
```

Comment:
@ValeriiMalkov It will see better with `typedef std::shared_ptr<NiceMock<MockApplication> > MockApplicationPtr` if you want